src/Microsoft.OData.Core/Json/ODataJsonCoreWriterUtils.cs

```diff
@@ -1,21 +1,20 @@
 ﻿//---------------------------------------------------------------------
-// <copyright file="ODataJsonLightWriterUtils.cs" company="Microsoft">
+// <copyright file="ODataJsonCoreWriterUtils.cs" company="Microsoft">
 //      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
 // </copyright>
 //---------------------------------------------------------------------
 
-namespace Microsoft.OData.JsonLight
+namespace Microsoft.OData.Json
 {
     #region Namespaces
     using System.Diagnostics;
     using System.Threading.Tasks;
-    using Microsoft.OData.Json;
     #endregion Namespaces
 
     /// <summary>
-    /// Helper methods used by the OData writer for the JsonLight format.
+    /// Helper methods used by the OData writer for the Json format.
     /// </summary>
-    internal static class ODataJsonLightWriterUtils
+    internal static class ODataJsonCoreWriterUtils

```

Comment:
I think I'd move the few methods that were in _ODataJsonLightWriterUtils.cs_ into _ODataJsonWriterUtils.cs_ instead introducing this _ODataJsonCoreWriterUtils.cs_ file. Or you could use _ODataJsonWriterExtension.cs_ (note that there's _JsonWriterExtensions_ file)

Reply:
Moved the methods to `ODataJsonWriterUtils.cs`